addons/events/XEH_postInit.sqf

```diff
@@ -11,3 +11,18 @@
         };
     } forEach allVariables GVAR(eventNamespaceJIP);
 }, []] call CBA_fnc_execNextFrame;
+
+
+if (isServer) then {
+    GVAR(clientID) = [0, 2] select isMultiplayer;
+    addMissionEventHandler ["PlayerConnected", {
+        params ["_id", "_uid", "_name", "_jip", "_owner"];
+        TRACE_5("PlayerConnected eh",_id,_uid,_name,_jip,_owner);
+
+        if (_owner != 2) then {
+            GVAR(clientID) = _owner;
+            _owner publicVariableClient QGVAR(clientID);
+            GVAR(clientID) = [0, 2] select isMultiplayer;
```

Comment:
Variable seems useful. Maybe name it "CBA_clientID" thereby make it "official" API, just like `CBA_missionTime` ?